flutter_local_notifications/lib/flutter_local_notifications.dart

```diff
@@ -32,6 +32,11 @@ export 'src/platform_specifics/ios/enums.dart';
 export 'src/platform_specifics/ios/initialization_settings.dart';
 export 'src/platform_specifics/ios/notification_attachment.dart';
 export 'src/platform_specifics/ios/notification_details.dart';
+export 'src/platform_specifics/linux/enums.dart';
+export 'src/platform_specifics/linux/icon.dart';
+export 'src/platform_specifics/linux/initialization_settings.dart';
+export 'src/platform_specifics/linux/method_channel_mappers.dart';
```

Comment:
"method_channel_mappers" sounds like an internal helper. is it intentionally exported?